ietf-layer0-types-ext.yang

```diff
@@ -157,7 +175,150 @@ module ietf-layer0-types-ext {
       "ITU-T G.698.2-201811 section 7 table 8-4/8-6";
   }
 
-// typedefs                
+  identity wavelength-assignment {

```

Comment:
For information : 
other drafts defining this identities are expired :
<pre><font color="#75507B">experimental/ietf-extracted-YANG-modules/ietf-te-wson-types@2018-08-15.yang</font>
<font color="#75507B">experimental/ietf-extracted-YANG-modules/ietf-wson-tunnel@2019-09-11.yang</font>
<font color="#75507B">experimental/ietf-extracted-YANG-modules/ietf-te-wson@2017-06-27.yang</font>
</pre>

and the following use the identy but refer to the wrong document (layer0)
<pre>
<font color="#75507B">experimental/ietf-extracted-YANG-modules/ietf-flex-grid-media-channel@2018-10-22.yang</font>
<font color="#75507B">experimental/ietf-extracted-YANG-modules/ietf-flexi-grid-media-channel@2019-03-24.yang</font>
</pre>